core-ui/src/shared/components/ResourceRef/ExternalResourceRef.js

```diff
@@ -1,53 +1,39 @@
 import React from 'react';
 import { useTranslation } from 'react-i18next';
-import { FormInput, ComboboxInput } from 'fundamental-react';
-
-import './ExternalResourceRef.scss';
+import { Dropdown } from 'react-shared';
 
 export function ExternalResourceRef({ resourceRef, onChange, resources }) {
-  const { t } = useTranslation();
-  const options = (resources || []).map((resource, index) => ({
-    key: index,
-    text: resource.metadata.name,
-    namespace: resource.metadata.namespace,
+  const { t, i18n } = useTranslation();
+
+  const format = resource => {
+    if (!resource) return null;
+    return `${resource.namespace}/${resource.name}`;
+  };
+
+  const options = (resources || []).map(resource => ({
+    key: format(resource.metadata),
+    text: format(resource.metadata),
+    resource,
   }));
 
   const selectResource = (e, selected) => {
-    if (selected.key > -1) {
-      onChange(e, { name: selected.text, namespace: selected.namespace });
-    }
+    onChange(e, {
+      name: selected.resource.metadata.name,
+      namespace: selected.resource.metadata.namespace,
+    });
   };
 
   return (
     <div className="external-resource-ref">
-      <ComboboxInput
+      <Dropdown
```

Comment:
has this change been consulted from the UX perspective?
as right now we end up with a VERY long unsearchable list of secrets. and no way to enter one manually.

Reply:
I'll move the `ResourceForm` to `shared` 👌 

About the `ExternalResourceRef`, this change came from functional review with Valentin, and the problem was there was no label for the namespace, but you're probably right, I'll roll it back.